app/templates/admin/userManagement.html

```diff
@@ -0,0 +1,65 @@
+{% extends "base.html" %}
+
+{% block scripts %}
+  {{super()}}
+  <script type="module" src="/static/js/userManagement.js"></script>
+
+{% endblock %}
+{% block app_content %}
+
+<h1 class="text-center">Admin Management</h1>
+
+{% macro createInputsButtons(inputId, buttonId, buttonName, buttonClass) %}
+<div class="input-group">
+  <input type="input" id="{{inputId}}" name="{{inputId}}" class="form-control" placeholder="Search" autocomplete="off"/>
+  <button type="button" class="btn btn-secondary btn-sm" disabled>
+    <span class="bi bi-search"></span>
+  </button>
+</div>
+<div class="col-md-4"><button type="button" class="btn btn-{{buttonClass}} btn-sm" id="{{buttonId}}">{{buttonName}}</button></div>
+{% endmacro %}
+<div class="container">
+<div class="row g-3">
+    <div class="col-md-4">
+      <div class="col-md-12">Add Celts Admin</div>
+        {{createInputsButtons("searchCeltsAdminInput","addCeltsAdmin","Add","success")}}
+    </div>
+  <div class="col-md-4 offset-md-4">
+    <div class="col-md-12">Add Celts Student Staff</div>
+    {{createInputsButtons("searchCeltsStudentStaffInput","addCeltsStudentStaff","Add","success")}}
+  </div>
+</div>
+ <br>
+  <div class="row g-3">
+  <div class="col-md-4">
+    <div class="col-md-12">Remove Celts Admin</div>
+    {{createInputsButtons("removeCeltsAdminInput","removeCeltsAdmin","Remove","danger")}}
+  </div>
+  <div class="col-md-4 offset-md-4">
+    <div class="col-md-12">Remove Celts Student Staff</div>
+    {{createInputsButtons("removeCeltsStudentStaffInput","removeCeltsStudentStaff","Remove","danger")}}
+  </div>
+</div>
+</div>
+
+<div class="row">
+  <div class="container-fluid mt-5 col-6">
+    <div class="text-center">
+      <h5 class="my-3">Select the Current Term</h5>
+      <div class="col-md-6 mx-auto" name="currentTermList" id="currentTermList">
+        {% for term in terms %}
+        <div class="row mx-auto">
+          <button class="btn btn-lg btn-outline-secondary text-start
+          {{ 'active' if selectedTerm.id == term.id }}" id="termFormID_{{term.id}}"
+          value='{{ term.id }}'>{{term.description}}</button>
+        </div>
+        {% endfor %}
+        <button class="btn btn-primary my-2 float-end"
+        id="submitButton" role="button">Make Current Term</button>
+      </div>
+    </div>
+  </div>
+  <input type="hidden" value="{{selectedTerm}}" id="termInput">
+  <input type="hidden" value="{{totalTerms}}" id="totalTerms">
```

Comment:
Rather than have to store the total terms to use later, you could just count the number of buttons in #currentTermList. That's pretty easy to do in js, and you don't have to manage a hidden field